### PR TITLE
v23.0.0 Update & Clean PR #3: Update test projects and examples

### PR DESCRIPTION
**Pull request by sachyco**
Refactored unit tests, e2e tests, and examples to use the updated client & transaction interfaces. Otherwise there are no functional changes or additional tests added.

Some key file changes are in the `source/Dgraph.tests.e2e/Tests/Approved/` folder.  Please review these to confirm that the updated schemas are correct. The new schemas were updated by running their respective tests and copying the received output. It is possible that the client or the unit tests have bugs that I do not have the dgraph expertise to diagnose.

### CODE REVIEW

**Comment by sachyco** [comment]
@mangalaman93 I recommend merging only this PR and closing the other two. That way the tests should pass, and that will complete Phase 1.

I can provide instructions for building and publishing the package to NuGet if necessary.

File: source/Dgraph.tests.e2e/Tests/Approved/SchemaTest.AlterSchemAsExpected.approved.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
 abool: bool .
+dgraph.drop.op: string .
+dgraph.graphql.p_query: string @index(sha256) .
 dgraph.graphql.schema: string .
+dgraph.graphql.xid: string @index(exact) @upsert .
```
Comment (MichelDiz) [inline source/Dgraph.tests.e2e/Tests/Approved/SchemaTest.AlterSchemAsExpected.approved.txt:5]: Have you tested push this schema? as far as I remember we can't use any predicate with prefix "dgraph.*".

**Review by MichelDiz — COMMENTED** [review]
The rest looks good to me. But let's wait Aman's meeting about those changes.

**Comment by mangalaman93** [comment]
@sachyco please rebase the PR on latest master

**Comment by sachyco** [comment]
> @sachyco please rebase the PR on latest master

@mangalaman93 Done

**Review by mangalaman93 — APPROVED** [review]

**Pull request merged**